lms/djangoapps/experiments/views.py

```diff
@@ -82,3 +81,25 @@ def bulk_upsert(self, request):
 
             serializer = self.get_serializer(upserted, many=True)
             return Response(serializer.data)
+
+
+class ExperimentKeyValueViewSet(viewsets.ModelViewSet):
+    authentication_classes = (JwtAuthentication, SessionAuthenticationAllowInactiveUser,)
```

Comment:
What authentication mechanism should we use from the jupyter notebook scripts?  

Since we want to make Bulk operations, we'll want to use a server-to-server type authentication, right?  So add OAuth here and allow Authorization Code grant?  

Or, should we ask each developer to enter their edx.org Staff account credentials when they connect via jupyter?

Regardless, wouldn't OAuth be easier to use via our script?